cves/CVE-2015-1872.yml

```
@@ -79,7 +90,7 @@ bugs_instructions: |
   Sometimes a commit message will mention a bug, or the security page on FFmpeg.
   Sometimes you need to search bug database, which is here:
       https://trac.ffmpeg.org
-bugs: []
+bugs: [ ]
```

```suggestion
bugs: []
```
I guess just leave this without the space in it.